### PR TITLE
Log client activity

### PR DESCRIPTION
**Pull request by ndowmon**
We know there is an issue in this step that prevents the integration from terminating. The changes here can help us answer the following questions:

1. Is there an issue when trying to fetch a new page of vulnerabilities?
2. Is there an issue when we try to find an existing vulnerability?

### CODE REVIEW

**Comment by ndowmon** [comment]
I want to test out adding CODEOWNERS to a project. Hopefully this auto-populates reviewers in PRs.

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (mknoedel) [inline CODEOWNERS:1]: Are we going to implement codeowners in all our projects?

**Review by mknoedel — DISMISSED** [review]
looks good so long as you are tracking this change somewhere so it can be reverted later.

File: CODEOWNERS (hunk 1 above)
Comment (ndowmon) [reply CODEOWNERS:1]: Maybe. We'll see how this one goes.

**Review by mknoedel — APPROVED** [review]

**Pull request merged**